extensions/rest/src/main/java/org/radargun/traits/RESTOperations.java

```diff
@@ -0,0 +1,23 @@
+package org.radargun.traits;
+
+import java.util.List;
+import javax.ws.rs.core.Cookie;
+import javax.ws.rs.core.MultivaluedMap;
+import org.radargun.Operation;
+import org.radargun.WrappedHttpResponse;
+
+/**
+ * @author Martin Gencur
+ */
+@Trait(doc = "Http operations.")
+public interface RESTOperations {
+   String TRAIT = RESTOperations.class.getSimpleName();
+
+   Operation GET = Operation.register(TRAIT + ".Get");
+
+   RESTOperationInvoker getRESTInvoker(String contextPath);
+
+   interface RESTOperationInvoker {
+      WrappedHttpResponse get(List<Cookie> cookies, MultivaluedMap<String, Object> headers);
```

Comment:
We could push this principle even to the level of returning `javax.ws.rs.client.Client`, but that's rather too much for our purpose, since we want to set the servers in configuration, not in the stage. So about the proper level could be:
```
interface RESTOperationInvoker {
     Invocation.Builder invoke(String contextPath);
}
```